Release-Notes/Release2.md

```diff
@@ -168,15 +168,15 @@ trademark certification.
 
 Our partner PlusServer has [achieved](https://www.openstack.org/brand/interop/)
 a [BSI C5](https://www.bsi.bund.de/EN/Topics/CloudComputing/Compliance_Criteria_Catalogue/Compliance_Criteria_Catalogue_node.html)
-security certification for their SCS implementation PlusCloud Open.
+security certification for their SCS implementation pluscloud open.
 
 We are working within [Gaia-X](https://gaia-x.eu/) to further the power
 of Gaia-X self-descriptions and are closely working with the
 [GXFS project](https://gxfs.de/)
-to jointly deliver a standard toolbox for Gaia-X conformant
+to jointly deliver a standard toolbox for Gaia-X conformance
```

Comment:
Maybe we should write `Gaia-X compliant infrastructure` ?

Reply:
yes, good one.